filebeat/prospector/log/prospector.go

```diff
@@ -509,3 +511,16 @@ func (l *Log) updateState(state file.State) error {
 
 	return nil
 }
+
+func (p *Prospector) Wait() {
+	p.registry.waitForCompletion()
+	p.Stop()
+}
+
+func (p *Prospector) Stop() {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported method Prospector.Stop should have comment or be unexported